python/observation-publisher/publisher/observability_backend.py

```diff
@@ -0,0 +1,47 @@
+import abc
+
+import pandas as pd
+from arize.pandas.logger import Client, Schema
+from arize.utils.types import Environments
+
+from publisher.config import (ArizeConfig, ModelSpec, ObservabilityBackend,
+                              ObservabilityBackendTypes)
+
+
+class ObservationSink(abc.ABC):
+    @abc.abstractmethod
+    def write(self, dataframe: pd.DataFrame):
+        raise NotImplementedError
+
+
+class ArizeSink(ObservationSink):
+    def __init__(self, config: ArizeConfig, model_spec: ModelSpec):
+        self._client = Client(space_key=config.space_key, api_key=config.api_key)
+        self._schema = Schema(
```

Comment:
the arize schema is depending on the model type, you need to check for each model type what are the required fields. For example for ranking model we need to specify `prediction_group_id_column_name` also it needs `rank_column_name` being set. You can refer to [this](https://docs.arize.com/arize/model-types/ranking) for ranking, and you can check for other model types